keyboards/meira/rules.mk

```diff
@@ -62,15 +62,15 @@ COMMAND_ENABLE ?= yes        # Commands for debug and configuration
 SLEEP_LED_ENABLE ?= no       # Breathing sleep LED during USB suspend
 # if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
 NKRO_ENABLE ?= no            # USB Nkey Rollover
-BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality on B7 by default
+BACKLIGHT_ENABLE ?= no       # Enable keyboard backlight functionality on B7 by default
```

Comment:
These `?=` shouldn't really be used anymore - did you have problems when using `=`? The rules.mk files should be included in a way that allow overriding.

Reply:
Oh, cool. I didn't know that. I just changed the default and noticed that it didn't match the others in the file.

Reply:
Should I go through and remove all the '?'s?

Reply:
That'd be awesome :) that way overriding works correctly everywhere - otherwise it's a bit confusing what's actually being enabled.